tests/chain_tests/plugins/tags/get_contents_tests.cpp

```diff
@@ -0,0 +1,100 @@
+#include <boost/test/unit_test.hpp>
+
+#include "tags_common.hpp"
+
+namespace get_contents_tests {
+
+using namespace scorum;
+using namespace scorum::tags;
+
+struct tags_get_contents_fixture : public database_fixture::tags_fixture
+{
+    tags_get_contents_fixture() = default;
+
+    struct content_query_wrapper : public api::content_query
```

Comment:
You could reorder member in api::content_query (move truncate_body to the end). In this case you won't need this wrapper

Reply:
It doesn't work in C++11 (we have C++14 only in develop branch)